### PR TITLE
Updated loadout: US_USMC_2010_wdl

### PR DESCRIPTION
**Pull request by AChesheireCat**
- Updated helmets to new CUP versions that don't look like poop
- Added microDAGR to items
- Fix'd optics for G/FTL/SL/FAC
- Fix'd attachments and bipod for AR
- Added laserbatteries to FAC
- Removed FIR dependencies for Helicopter and Jet Pilots
- Updated HMG to use ACE CSW
- Changed inheritance of vehicle crews from SMG to CAR for period accuracy
- Probably some other minor stuff I missed lol

### CODE REVIEW

**Comment by Freddo3000** [comment]
Protip: If you put `resolves/closes/fixes #90` in the description, then the ticket will be automatically closed when the PR is merged.

File: addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp [hunk 1]
```diff
@@ -25,75 +24,62 @@ class baseMan {// Weaponless baseclass
...
-	// Leave empty to remove all. "Default" for secondaryWeapon or sidearmWeapon > leave original weapon.
 
-	// These are added to the uniform or vest
 	magazines[] = {};
 	items[] = {
 		 MEDICAL_R ,
 		"ACE_Flashlight_XL50",
-		"ACE_SpraypaintGreen"
+		"ACE_SpraypaintGreen",
+		"ACE_microDAGR"
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp:46]: MicroDAGR was just about starting to be fielded in 2010: https://gisuser.com/2009/10/rockwell-collins-new-microdagr-gps-receiver-for-soldiers-features-latest-commercial-apps/

File: addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp [hunk 2]
```diff
@@ -187,15 +176,16 @@ class sl : ftl {
 
 class co : sl {
 	displayName = "Platoon Leader";
-	headgear[] = {"CUP_H_USMC_CAP_PRR_DES"};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp:190]: muh hat

File: addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp [hunk 3]
```diff
@@ -187,15 +176,16 @@ class sl : ftl {
...
 	vest[] = {"CUP_V_B_Eagle_SPC_Officer"};
 };
 
 class fac : co {
 	displayName = "Forward Air Controller";
 	vest[] = {"CUP_V_B_Eagle_SPC_RTO"};
-	backpack[] = {"CUP_B_Kombat_Radio_Olive"};
+	backpack[] = {"B_Kitbag_cbr"};
+	hmd[] = {"antenna_long_two_nvg"};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp:186]: Unlisted dependency

File: addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp [hunk 4]
```diff
@@ -187,15 +176,16 @@ class sl : ftl {
...
 	backpackItems[] = { FAC_GEAR };
+	magazines[] = {"Laserbatteries"};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp:188]: Overwriting magazines

File: addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp [hunk 5]
```diff
@@ -291,20 +282,32 @@ class mmgtl : aar {
...
-	backPack[] = {"CUP_B_M2_Tripod_Bag"};
+	backPack[] = {"B_Carryall_cbr"};
+	secondaryWeapon[] = {"ace_csw_m3CarryTripodLow"};
+	backpackItems[] = {
+		LIST_3("ace_csw_100Rnd_127x99_mag_red")
+	};
+	items[] += {"ACE_MapTools"};
 	linkedItems[] += {
-		"ACE_Vector",
-		"ItemGPS"
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp:307]: See note regarding MicroDAGR

**Review by Freddo3000 — CHANGES_REQUESTED** [review]

File: addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp (hunk 1 above)
Comment (AChesheireCat) [reply addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp:46]: Fix'd by only giving it to SLs and above.

File: addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp (hunk 2 above)
Comment (AChesheireCat) [reply addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp:190]: xD

File: addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp (hunk 3 above)
Comment (AChesheireCat) [reply addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp:186]: Fix'd

File: addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp (hunk 4 above)
Comment (AChesheireCat) [reply addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp:188]: Fix'd

File: addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp (hunk 5 above)
Comment (AChesheireCat) [reply addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp:307]: Fix'd

**Review by Freddo3000 — APPROVED** [review]
Just going to assume you've run this through the autotest at least

**Comment by AChesheireCat** [comment]
Caught one last thing that I noticed in the russian loadouts as well; the MTR's items[] = {} didn't have the += so it was overwriting their medical stuff. Fix'd and nothing else was touched.

**Pull request merged**